### PR TITLE
feat(@desktop/wallet): add support for approve and swap activity entries

### LINKED ISSUE #14836: Activity List
https://www.notion.so/Activity-List-6c477643054645109b82a6da29d99d68
https://www.figma.com/design/TS0eQX9dAZXqZtELiwKIoK/Swap---Milestone-1?node-id=3541-467934&t=eTvhBjNa4YYvdRU7-0

Handle Approve Tx type

### PR DESCRIPTION
**Pull request by dlipicar**
Fixes #14836
status-go part: https://github.com/status-im/status-go/pull/5530

### What does the PR do

Temporary solution to get activty entries for Approve and Swap transactions. Full implementation to be done under https://github.com/status-im/status-desktop/issues/15621, I just didn't manage to finish it in time.

image

### CODE REVIEW

File: src/app_service/common/wallet_constants.nim [hunk 1]
```diff
@@ -9,3 +9,5 @@ const
 
   TX_HASH_LEN* = 32 * 2
   TX_HASH_LEN_WITH_PREFIX* = TX_HASH_LEN + 2
+
+  PARASWAP_V5_CONTRACT_ADDRESS* = "0x1b40183efb4dd766f11bda7a7c3ad8982e998421" # Same address for all supported chains
```
Comment (Khushboo-dev-cpp) [inline src/app_service/common/wallet_constants.nim:13]: I guess this only till we have a list from backed. lets add a TODO with link to https://github.com/status-im/status-desktop/issues/15329 ?

File: ui/StatusQ/src/assets.qrc [hunk 2]
```diff
@@ -324,7 +325,6 @@
         <file>assets/img/icons/username.svg</file>
         <file>assets/img/icons/view.svg</file>
         <file>assets/img/icons/wallet.svg</file>
-        <file>assets/img/icons/walletConnect.svg</file>
```
Comment (Khushboo-dev-cpp) [inline ui/StatusQ/src/assets.qrc:327]: why was this removed?

**Review by Khushboo-dev-cpp — APPROVED** [review]
LGTM !

**Review by caybro — APPROVED** [review]
LGTM

Can you pls also use the newly introduced Constants in SwapModal? There's a bunch of `FIXME`s like:
```qml
// serviceProviderURL: "" // FIXME get the service provider URL from backend
// serviceProviderIcon: "" // FIXME get the service icon from backend
```

**Comment by virginiabalducci** [comment]
Looks good! QA passed @dlipicar
